### PR TITLE
Promote Managed SSL to GA + Doc fixes

### PR DESCRIPTION
**Pull request by upodroid**
Fixes: https://github.com/hashicorp/terraform-provider-google/issues/7828
Fixes: https://github.com/hashicorp/terraform-provider-google/issues/7832

If this PR is for Terraform, I acknowledge that I have:

- [X] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [X] [Generated Terraform](https://github.com/GoogleCloudPlatform/magic-modules#generating-downstream-tools), and ran `make test` and `make lint` to ensure it passes unit and linter tests.
- [X] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/master/third_party/terraform/tests) (for handwritten resources or update tests).
- [X] [Ran](https://github.com/hashicorp/terraform-provider-google/blob/master/.github/CONTRIBUTING.md#tests) relevant acceptance tests (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [X] Read the [Release Notes Guide](https://github.com/GoogleCloudPlatform/magic-modules/blob/master/.ci/RELEASE_NOTES_GUIDE.md) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:enhancement
compute: promoted `google_compute_managed_ssl_certificate` to GA
```

### CODE REVIEW

**Review by c2thorn — CHANGES_REQUESTED** [review]
Moving this to GA caused MM to try to generate this for Ansible. I believe it needs to be excluded here: 
https://github.com/GoogleCloudPlatform/magic-modules/blob/master/products/compute/ansible.yaml#L327

**Comment by upodroid** [comment]
Ah, that explains why generate-diffs was broken

**Review by c2thorn — CHANGES_REQUESTED** [review]

File: products/compute/ansible.yaml [hunk 1]
```diff
@@ -61,6 +61,8 @@ datasources: !ruby/object:Overrides::ResourceOverrides
     exclude: true
   RegionPerInstanceConfig: !ruby/object:Overrides::Ansible::ResourceOverride
     exclude: true
+  ManagedSslCertificate: !ruby/object:Overrides::Ansible::ResourceOverride
```
[the remark this answers is not in the document]
Comment (upodroid) [reply products/compute/ansible.yaml]: Oopsie, ficed

**Review by c2thorn — CHANGES_REQUESTED** [review]
https://github.com/GoogleCloudPlatform/magic-modules/blob/master/third_party/terraform/utils/common_diff_suppress.go.erb#L140 needs to be made available for GA as well, the build is failing because of this

**Comment by upodroid** [comment]
I fixed and tested it properly

**Review by c2thorn — APPROVED** [review]

**Comment by c2thorn** [comment]
@upodroid thanks for your patience on this, but I think we are at a good point now.

**Pull request merged**